### DISCUSSION TITLE
Janus graph keeps restarting

### DISCUSSION THREAD

**Discussion by vishnuamritanshuis** [JanusGraph/janusgraph#3316] [Q&A]
Hi, 
In our web API app, we use Janusgraph DB( storage Berkeley) as the backend. It generally works fine, but for one of our edge machines, the DB keeps restarting repeatedly. 
When checking the logs it seems like the issue is to Diskspace size but when checking the available space it has enough space for the Janus graph to work. Please check the attached image for details. 
![image](https://user-images.githubusercontent.com/49487451/201100851-8e1a3094-616b-4e96-b027-74942a1328b4.png).

I am attaching the logs, any advice 
[logs.txt](https://github.com/JanusGraph/janusgraph/files/9980963/logs.txt)
is highly appreciated.

**Comment by FlorianHockmann** [comment]
The log contains a lot of `Permission denied` errors that shouldn't be there. Maybe that's also what's really causing the exception:
_Disk usage is not within je.maxDisk or je.freeDisk limits **and write operations are prohibited**_
